* Fix PYINCDIR by always using 'posix_prefix', otherwise some other scheme may
  be used depending on the platform.
* Don't define a "module" extension for python2 - this breaks PLIST and loading a
  bare .so is supported anyway by python2. python3 has SOABI, or rather
  EXT_SUFFIX nowadays.

--- mk/python.mk.orig	2023-07-13 12:38:54.000000000 +0200
+++ mk/python.mk	2024-06-20 13:35:01.491768788 +0200
@@ -4,11 +4,11 @@
 
 PYVERSION := $(shell $(PYTHON) -c 'import sys; sys.stdout.write(".".join(sys.version.split(".")[:2]))')
 PYPREFIX  := $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.exec_prefix.replace("\\","/").replace(" ","\\ "))')
-PYINCDIR  := $(shell $(PYTHON) -c 'import sys, sysconfig; sys.stdout.write(sysconfig.get_path("include").replace("\\","/").replace(" ","\\ "))')
+PYINCDIR  := $(shell $(PYTHON) -c 'import sys, sysconfig; sys.stdout.write(sysconfig.get_path("include", "posix_prefix").replace("\\","/").replace(" ","\\ "))')
 
-PythonSHAREDLIB_SUFFIX = $(shell $(PYTHON) -c 'import sys, sysconfig; sys.stdout.write((sysconfig.get_config_var("SO") or ".so").lstrip("."))')
+PythonSHAREDLIB_SUFFIX = $(shell $(PYTHON) -c 'import sys, sysconfig; sys.stdout.write((sysconfig.get_config_var("EXT_SUFFIX") or ".so").lstrip("."))')
 
-PY_MODULE_SUFFIX := $(shell $(PYTHON) -c 'import sys; sys.stdout.write((sys.hexversion < 0x3000000 and not hasattr(sys, "pypy_version_info")) and "module" or "")')
+PY_MODULE_SUFFIX :=
 
 PYINCFILE := "<Python.h>"
 PYINCTHRD := "<pythread.h>"
